src/AppBundle/Resources/views/Pa/Team/addUser.html.twig

```diff
@@ -0,0 +1,31 @@
+{% extends 'AppBundle:Layouts:application.html.twig' %}
+
+{% import 'AppBundle:Macros:macros.html.twig' as macros %}
+
+{% set translationDomain = "user-details" %}
+{% trans_default_domain translationDomain %}
+
+{% block htmlTitle %}{{ 'htmlTitle.add-account' | trans({}, 'user-account') }}{% endblock %}
+{% block pageTitle %}{{ 'h1Title.add-account' | trans({}, 'user-account') }}{% endblock %}
+
+{% block breadcrumbs %}
+    {{ macros.breadcrumbsSettings('user-add') }}
+{% endblock %}
+
+{% block pageContent %}
+    {{ form_start(form, {attr: {novalidate: 'novalidate' }}) }}
+
+    {% include 'AppBundle:User:_form_fields.html.twig' %}
+
+    {{ form_submit(form.save,'form.editYourDetails.controls.save', {'buttonClass': 'behat-link-save'}) }}
```

Comment:
not sure about using editYourDetails. However im unsure where/how these controls are set?